packages/universal-router-sdk/src/utils/constants.ts

```diff
@@ -0,0 +1,139 @@
+import { ChainId } from '@pancakeswap/chains'
+import { Address } from 'viem'
+import { ChainConfig } from './types'
+
+const WETH_NOT_SUPPORTED_ON_CHAIN = '0x0000000000000000000000000000000000000000'
+
+const CHAIN_CONFIGS: { [key: number]: ChainConfig } = {
```

Comment:
Can we simply have `UNIVERSAL_ROUTER_ADDRESSES` and `PERMIT2_ADDRESSES`? I don't think we need weth address here. And what is the creationBlock for?

Reply:
yep removed weth and creation block from config and use WETH9 instead